### DISCUSSION TITLE
Settings issues on site update

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1474] [Q&A]
I am working in a staging site and I have two issues I am hoping someone can offer some guidance on. First I am having issues with the upcoming meetings widget in my sidebar. The font size with my new theme Astra is not allowing me to adjust font size for that widget and the text is breaking poorly, you see what I mean on this page, https://008.681.myftpupload.com/al-anon-links/. I do not see anywhere where I can control the appearance of the table in settings? It also is coming up as a legacy widget with not much other info.

The other issue is I wanted to switch the the TSML UI but when I do that the table goes into a 2nd column that I don't understand because it's a full width page. I have a capture of the differences attached here
![meeting-interface-issue](https://github.com/user-attachments/assets/81ac0c19-f0ed-4d16-9db5-2294228c94c7)

I am not the original builder of the site but I  do not see any place to customize this. We use elementor and I can't make changes to the meeting table with it.

Any ideas are greatly appreciated

Thanks
Rose L
Al-anon
District 6 NC/Bermuda

**Comment by thessnake03** [comment]
The TSML_UI design replaces the need for the Upcoming Widget. If drop the 2nd column and you should be good to go.

If you are going to use the sidebar, try putting the shortcode {tsml_ui} in the widget. Could also try restricting the use of all the default column by inserting some code in the child theme, or tweak the specific one in css.

I believe in Astra you can use Meetings Main Sidebar as well.

If you still want font tweaks after replacing the widget, css would be what I would do. 

Hope that helps.

**Comment by district6wc** [comment]
It looks like with the TSML UI you cannot have anything else on the page? Also in legacy the table just displayed the day you landed on the page, so Thursday would be there if you went on Thursday, Friday if you went on Friday, Can the TSML UI be set up like that?

**Comment by thessnake03** [comment]
My experience with tsml ui (and legacy), it's easier to customize a page that isn't /meetings and point to that. Like we've done with https://code4recovery.org/tsml-ui

Tsml ui should not only jump to today, but also the current time to show meetings in progress.

**Comment by district6wc** [comment]
Hi [thessnake03] that is not a sidebar it is supposed to be below the table. I had a few more questions I added. Hoping someone will se that but I tried the short code but did not seem to work for me

**Comment by joshreisner** [comment]
looks like the page is showing up in columns because of some styling coming from the theme, not TSML UI. there is a div with class `ast-container` which is setting the layout here to be flex, here's some some custom CSS but i think that something might not be set up correctly? not sure why they would want it like that.

`body.tsml-ui .ast-container { flex-direction: column; }`

<img width="1840" alt="Screenshot 2024-07-25 at 3 06 15 PM" src="https://github.com/user-attachments/assets/69a1879e-7192-44fa-b271-9e3963762653">

as for the upcoming meetings widget, if you go into properties and click the "Style with CSS?" box it should look a little better for you

<img width="168" alt="Screenshot 2024-07-25 at 3 10 02 PM" src="https://github.com/user-attachments/assets/1b996074-ea76-4305-8c49-a36e8883b515">

**Comment by district6wc** [comment]
Hi

I rebuilt the page and the short code is working I do this something corrupted on my original page, since I use the Upcoming meeting in the sidebar on other pages I'll see what I can do with css to get those fonts to render a little better.

Can you set the TSML_UI to just show current day of the week? I also see it is pulling in all my inactive meetings the legacy version did not do that.

I really appreciate the help you have provided

Roe

**Reply by joshreisner** [reply]
TSML UI shows the next meeting chronologically, because this is a better experience for users. for example, if it showed the current day it would show only meetings in the past, since the last meeting today is already over.

however at vritually every time of day, the legacy interface forced users to scroll past inactive meetings. this is especialy bad as the great majority of users are on a mobile device. TSML UI puts the next meetings at the top of the screen, and we hear from users that they greatly prefer this UX

it would be good to remove the inactive meetings from the database. looks like there are only three though https://008.681.myftpupload.com/test-shortcode-for-meetings/?type=inactive

**Comment by district6wc** [comment]
Josh, I found some short code on here so I could rebuild the sidebar element and ran some style in a div tag that helped me control font-size and weight n that that table but the table heads are still breaking, 

![image](https://github.com/user-attachments/assets/12f313eb-36da-4e5a-a198-258c80796430)

our old site (using the Legacy UI was using much larger font-size and I dd not have this issue
![image](https://github.com/user-attachments/assets/d8f681cb-ad72-49d7-891a-083518a3419b)

Is there any ability to control this. We are using the Astra theme with elementor

Thanks so much
Rose

**Reply by district6wc** [reply]
Just thinking about another option to not display city, this is the shortcode I am using tsml_next_meetings count="3" Is there a short code that could do that?

**Reply by joshreisner** [reply]
this word-break style is also coming from the Astra theme. no idea why they would want that

try adding this to your CSS: `table.tsml_next_meetings { word-break: normal; }` - it should override Astra's secondary column css

the city adds essential information for users so i wouldn't hide that

**Comment by district6wc** [comment]
Thank you, I added it to several css files nothing, then added it inline and it works so I am grateful
